core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneOptimized.java

```diff
@@ -125,6 +130,26 @@ private DaoGeneOptimized () {
         }
     }
 
+    private void clearCache()
```

Comment:
for the unit tests. Different unit tests use different set of genes. Some Import*.java classes could have scenarios where a specific gene is missing, or specific aliases are added or removed, etc